webapp/app/forms/fields.py

```diff
@@ -78,7 +78,7 @@ def __call__(self, field, **kwargs):
 
         if 'required' not in kwargs and 'required' in getattr(field, 'flags', []):
             kwargs['required'] = True
-        kwargs['class'] = 'form-control'
+        kwargs['class'] = kwargs.get('class', '') + ' form-control'
```

Comment:
If no class is set, this results in `' form-control'` (with initial space), which we don't want. I also don't like that this is different from how adding a class is implemented in the IBAN field.

Suggestion: factor out a `_add_classes_to_kwargs(kwargs)` module-level function and use it in all places where fields add classes:

```py
def _add_classes_to_kwargs(kwargs, classes):
    joined_classes = ' '.join(classes)
    if 'class' in kwargs:
        kwargs['class'] += ' ' + joined_classes
    else:
        kwargs['class'] = joined_classes
```
(ohne Gewähr)

Reply:
I agree. I also did not really like the initial space to begin with but I did not want to add more conditionals. So, adding the function is a great idea.